borsh/tests/test_schema_enums.rs

```diff
@@ -148,6 +148,7 @@ pub fn complex_enum_with_schema() {
     // Then check that we serialize and deserialize with schema.
     let obj = A::default();
     let data = try_to_vec_with_schema(&obj).unwrap();
+    // insta::assert_debug_snapshot!(buf);
```

Comment:
Why is it commented out? Is it too big? Is it not stable?

Reply:
I've mentioned that this example of schema derivation https://github.com/near/borsh-rs/blob/master/borsh-schema-derive-internal/src/enum_schema.rs#L365 is suboptimal, as structs representing enum variants depend on more generic params than their counterparts in original enum. 
There may be more of such details, which will affect schema serialization.